### PR TITLE
include hxhash data in getTokenWithdrawEventData; add missing return

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

File: packages/arb-ts/src/lib/l2Bridge.ts [hunk 1]
```diff
@@ -184,6 +187,7 @@ export class L2Bridge {
           contract: arbERC20TokenContract,
           balance,
         }
+        return tokenData
```
Comment (hkalodner) [inline packages/arb-ts/src/lib/l2Bridge.ts:190]: Don't we want to keep going so we can also potentially fill in ERC-777 data?

**Review by hkalodner — APPROVED** [review]
LGTM

File: packages/arb-ts/src/lib/l2Bridge.ts [hunk 2]
```diff
@@ -223,10 +223,13 @@ export class L2Bridge {
       )
       const balance = await arbERC777TokenContract.balanceOf(walletAddress)
       tokenData.ERC777.balance = balance
+    }
+    if (tokenData.ERC20 || tokenData.ERC777 || tokenData.CUSTOM) {
       return tokenData
```
Comment (DZGoldman) [inline packages/arb-ts/src/lib/l2Bridge.ts:228]: this gets all token types and preserves the return semantics we want

File: packages/arb-ts/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "arb-ts",
-  "version": "0.0.2",
+  "version": "0.0.3",
```
Comment (fredlacs) [inline packages/arb-ts/package.json:3]: Is it `v0.0.4` now?

**Pull request merged**